src/tutorial/microbe_editor/NucleusTutorial.cs

```diff
@@ -0,0 +1,106 @@
+﻿namespace Tutorial;
+
+using System;
+using Newtonsoft.Json;
+
+/// <summary>
+///   Prompts the player to eventually place a nucleus for further progression
+/// </summary>
+public class NucleusTutorial : EditorEntryCountingTutorial
+{
+    private readonly string cellEditorTab = EditorTab.CellEditor.ToString();
+
+    private readonly OrganelleDefinition nucleus = SimulationParameters.Instance.GetOrganelleType("nucleus");
+
+    [JsonProperty]
+    private bool hasNucleus;
+
+    public NucleusTutorial()
+    {
+        CanTrigger = false;
+    }
+
+    public override string ClosedByName => "NucleusTutorial";
+
+    protected override int TriggersOnNthEditorSession => 10;
+
+    public override void ApplyGUIState(MicrobeEditorTutorialGUI gui)
+    {
+        gui.NucleusTutorialVisible = ShownCurrently;
+    }
+
+    public override bool CheckEvent(TutorialState overallState, TutorialEventType eventType, EventArgs args,
+        object sender)
+    {
+        if (!hasNucleus)
+        {
+            if (base.CheckEvent(overallState, eventType, args, sender))
+                return true;
+        }
+
+        switch (eventType)
+        {
+            case TutorialEventType.MicrobeEditorOrganellePlaced:
+            {
+                if (args is not OrganellePlacedEventArgs eventArgs)
```

Comment:
Like the chemoreceptor tutorial I don't think this should do a safe cast, better to fail loudly if someone messes up MicrobeEditor**OrganellePlaced** event.